facebook/delphiFacebook/R/variables.R

```diff
@@ -646,8 +646,8 @@ code_vaccines <- function(input_data, wave) {
     # If the entire column is NA, ifelse() results in a logical vector, not a
     # character vector, which confuses split_options; since the result should be
     # NA anyway
+    vaccine_barriers <- ifelse(vaccine_barriers == "13", NA_character_, vaccine_barriers)
     if (any(!is.na(vaccine_barriers))) {
-      vaccine_barriers <- ifelse(vaccine_barriers == "13", NA_character_, vaccine_barriers)
```

Comment:
if the entire column is NA, and (thus) ifelse() results in a logical vector, how does that affect `any(!is.na(vaccine_barriers))`?

Reply:
`is.na` appears to have the same behavior for logical `NA` and character missing (`NA_character_`), but I'll explicitly cast to character to be safe. The downstream `is_selected` also appears to work correctly with logical `NA` but expects strings.